tests/pipelines/text_to_video/test_text_to_video_zero_sdxl.py

```diff
@@ -0,0 +1,108 @@
+# coding=utf-8
+# Copyright 2023 HuggingFace Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+import torch
+
+from diffusers import DDIMScheduler, TextToVideoZeroSDXLPipeline
+from diffusers.utils import require_torch_gpu, slow, torch_device
+
+
+class TextToVideoZeroSDXLPipelineFastTests(unittest.TestCase):
+    def test_forward_loop(self):
+        model_id = "stabilityai/stable-diffusion-xl-base-1.0"
```

Comment:
You should create a dummy model for the fast test instead of using a pre-trained checkpoint. Maybe you can take a look at the sdxl test as an example https://github.com/huggingface/diffusers/blob/5c404f20f4462bc07669280c9b9126a10196a34a/tests/pipelines/stable_diffusion_xl/test_stable_diffusion_xl.py#L52

Reply:
cc @DN6 here

Reply:
Agree with @yiyixuxu